pandas/io/parsers/base_parser.py

```diff
@@ -202,6 +208,25 @@ def __init__(self, kwds):
 
         self.handles: Optional[IOHandles] = None
 
+        # Bad line handling
+        on_bad_lines = kwds.get("on_bad_lines")
+        if on_bad_lines is not None:
+            if on_bad_lines == "error":
+                self.on_bad_lines = self.BadLineHandleMethod.ERROR
+            elif on_bad_lines == "warn":
+                self.on_bad_lines = self.BadLineHandleMethod.WARN
+            elif on_bad_lines == "skip":
+                self.on_bad_lines = self.BadLineHandleMethod.SKIP
+            else:
+                raise ValueError(f"Argument {on_bad_lines} is invalid for on_bad_lines")
+        else:
+            if kwds.get("error_bad_lines"):
```

Comment:
these need a deprecation warning

Reply:
_deprecated_defaults handles this for us.